src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormulaIT.kt

```diff
@@ -14,13 +14,19 @@ import com.atlassian.performance.tools.workspace.api.TaskWorkspace
 import com.atlassian.performance.tools.workspace.api.TestWorkspace
 import com.google.common.util.concurrent.ThreadFactoryBuilder
 import org.apache.logging.log4j.CloseableThreadContext
-import org.assertj.core.api.Assertions.assertThat
```

Comment:
We usually migrate away from Hamcrest and towards AssertJ.

Reply:
Done.